contrib/completion/bash/docker

```diff
@@ -469,6 +480,7 @@ __docker_tasks() {
 }
 
 # __docker_complete_services_and_tasks applies completion of services and task IDs.
+# shellcheck disable=SC2120
```

Comment:
It could definitely be reduced by having a global disable on a file basis. It's a trade-off with granularity though.